e2e/nomostest/wait_for_sync.go

```diff
@@ -476,7 +476,9 @@ func (nt *NT) WaitForRepoSyncStalledError(rsNamespace, rsName, reason, message s
 // 	return nil
 // }
 
-func validateError(errs []v1beta1.ConfigSyncError, code string, message string) error {
+// ValidateConfigSyncError returns true if the specified errors contain an error
+// with the specified error code and (partial) message.
+func ValidateConfigSyncError(errs []v1beta1.ConfigSyncError, code string, message string) error {
```

Comment:
Why the export and name change?

Reply:
Wanted to use it for custom predicates. Tho it turns out i didn't need to after I moved the predicates into the nomostest package. I could change it back...

Reply:
Might as well not change it if there's no need 🤷

Reply:
reverted